node/smeshing_service.go

```diff
@@ -479,41 +421,14 @@ func (app *App) initActivationServiceServices(ctx context.Context) error {
 	}
 
 	app.mesh = msh
-	app.syncer = newSyncer
 	app.atxBuilder = atxBuilder
 	app.atxHandler = atxHandler
 	app.poetDb = poetDb
-	app.fetcher = fetcher
-	app.beaconProtocol = beaconProtocol
 
 	return nil
 }
 
-func (app *App) startActivationServiceServices(ctx context.Context) error {
-	if app.fetcher != nil {
-		if err := app.fetcher.Start(); err != nil {
-			return fmt.Errorf("start fetcher: %w", err)
-		}
-	}
-	if app.syncer != nil {
-		app.syncer.Start()
-	}
-
-	if app.beaconProtocol != nil {
-		app.beaconProtocol.Start(ctx)
-	}
-
-	if app.blockGen != nil {
-		app.blockGen.Start(ctx)
-	}
-	if app.certifier != nil {
-		app.certifier.Start(ctx)
-	}
-	if app.proposalBuilder != nil {
-		app.eg.Go(func() error {
-			return app.proposalBuilder.Run(ctx)
-		})
-	}
+func (app *App) startSmeshingServiceServices(ctx context.Context) error {
 	if app.remoteProposalBuilder != nil {
```

Comment:
It's an error when `remoteProposalBuilder` is nil. Consider: 
```go
if app.remoteProposalBuilder == nil {
    return errors.New("proposal builder cannot be nil")
}

Reply:
That should newer happend. I'll use `panic` instead of returning an error there.

Reply:
Done.